src/status_im/profile/handlers.cljs

```diff
@@ -13,28 +13,31 @@
   (when identity
     (dispatch [:navigation-replace :chat identity])))
 
-(register-handler :open-image-picker
-   (u/side-effect!
-     (fn [_ _]
-       (show-image-picker
-         (fn [image]
-           (let [path       (get (js->clj image) "path")
-                 _ (log/debug path)
-                 on-success (fn [base64]
-                              (dispatch [:set-in [:profile-edit :photo-path] (str "data:image/jpeg;base64," base64)]))
-                 on-error   (fn [type error]
-                              (.log js/console type error))]
-             (img->base64 path on-success on-error)))))))
-
-(register-handler :phone-number-change-requested
+(register-handler
+  :open-image-picker
+  (u/side-effect!
+    (fn [_ _]
+      (show-image-picker
+        (fn [image]
+          (let [path       (get (js->clj image) "path")
+                _ (log/debug path)
+                on-success (fn [base64]
+                             (dispatch [:set-in [:profile-edit :photo-path] (str "data:image/jpeg;base64," base64)]))
+                on-error   (fn [type error]
+                             (.log js/console type error))]
+            (img->base64 path on-success on-error)))))))
+
+(register-handler
+  :phone-number-change-requested
   ;; Switch user to the console issuing the !phone command automatically to let him change his phone number.
   ;; We allow to change phone number only from console because this requires entering SMS verification code.
   (u/side-effect!
     (fn [db _]
       (dispatch [:navigate-to :chat console-chat-id])
       (js/setTimeout #(dispatch [:select-chat-input-command {:name "phone"}]) 500))))
 
-(register-handler :open-chat-with-the-send-transaction
+(register-handler
+  :open-chat-with-the-send-transaction
```

Comment:
+1 for establishing a convention. `cursive` was happier with 2 lines but I'm fine with both.